doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_architecture_ipcr.rst

```diff
@@ -1,5 +1,3 @@
-:orphan:
-
 .. _ug_nrf54h20_architecture_ipc:
```

Comment:
@KAGA164 Hey Kamil, could you check if this document is still valid?

Reply:
or @doki-nordic